python-main.js

```diff
@@ -250,7 +250,7 @@ function web_editor(config) {
             EDITOR.focus();
         } else {
             // If there's no name, default to something sensible.
-            setName("microbit");
+            setName("my program");
```

Comment:
@jaustin I agree with Daniel here that we should have a more explicity script name than "micro:bit". Otherwise it would say "microbit" in this view:
![image](https://user-images.githubusercontent.com/29712657/58638895-b65e3c80-82ed-11e9-8b67-aae5b16311bb.png)

Not sure if "my program" (what we used in the whiteboard) is the best option, what do you think?

Reply:
Chatting with Jonny we think "micro:bit program" is a good alternative here.